.github/workflows/build_and_test.yml

```diff
@@ -258,6 +263,12 @@ jobs:
           profile: minimal
           target: x86_64-apple-ios
 
+      - name: Install cbindgen
+        # FIXME: Wait until there's a released version of cbindgen!
+        run: cargo install cbindgen --git https://github.com/eqrion/cbindgen
+        env:
+          CARGO_TARGET_DIR: cargo-install-cache
```

Comment:
Did not work the way I wanted it to: https://github.com/signalapp/libsignal-client/pull/122/checks?check_run_id=1572842893